src/Sylius/Behat/Context/Ui/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -61,6 +66,11 @@ public function __construct(
         $this->indexPage = $indexPage;
         $this->updatePage = $updatePage;
         $this->currentPageResolver = $currentPageResolver;
+        $this->gatewayFactories = [
```

Comment:
I'd love to, but I don't know exactly how - this is used in context of two different types (**setup** and **ui**), therefore this array should be defined as parameter it the `contexts.xml` file - which wouldn't look good. However, I've injected this array here and in `PaymentContext`, as you proposed in other comment - I think it's the most proper level of abstraction in this case.